Update link of Create copy of VHD stored as a Managed Disk using Snap…

laraaleite opened the pull request:
…shots in Linux

Update the following link (is currently linking to Windows article instead of Linux):
* [Create copy of VHD stored as a Managed Disk using Snapshots in Linux](../articles/virtual-machines/linux/snapshot-copy-managed-disk.md)

azurecla commented:
Hi __@seanfreiburg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jimbe commented:
Sean - looks like the broken link fix is simple enough. But your pull request has merge conflicts. Can you resolve them? Thanks.

tysonn commented:
Sent mail

tysonn commented:
Sean, could you please resolve the merge conflicts and remove the /en-us/ part of the path you added to a link? That section breaks the localized experience by forcing users from a translated article back to the EN version of the article.

tysonn commented:
Closing since merge conflict has not been resolved.

laraaleite committed:
Update link of Create copy of VHD stored as a Managed Disk using Snapshots in Linux

Update the following link (is currently linking to Windows article instead of Linux):
* [Create copy of VHD stored as a Managed Disk using Snapshots in Linux](../articles/virtual-machines/linux/snapshot-copy-managed-disk.md)
